go/checks/checks.go

```diff
@@ -163,3 +163,11 @@ func CheckMaxPartitionsContributed(label string, maxPartitionsContributed int64)
 	}
 	return nil
 }
+
+// CheckConfidenceLevel returns an error if the supplied confidence level is not between 0 and 1
+func CheckConfidenceLevel(label string, confidenceLevel float64) error {
+	if confidenceLevel < 0 || confidenceLevel > 1 || math.IsNaN(confidenceLevel) || math.IsInf(confidenceLevel, 1) || math.IsInf(confidenceLevel, -1) {
```

Comment:
math.IsInf(confidenceLevel, 0) should check whether it is any infinity

Reply:
@MashaTelyatnikova  Is it necessary to change the two tests to only one test?

Reply:
1. it is shorter, Go idiom I guess is to short everything is possible
2. it is consistent with other checks (see example https://github.com/google/differential-privacy/blob/main/go/checks/checks.go#L97)